vsdkx/addon/tracking/processor.py

```diff
@@ -16,6 +20,8 @@ def __init__(self, addon_config: dict, model_settings: dict,
                  model_config: dict, drawing_config: dict):
         super().__init__(addon_config, model_settings, model_config,
                          drawing_config)
+        self._logger = logging.getLogger(LOG_TAG)
```

Comment:
Can we inherit the logging level from one global setting and have it affect all loggers in all drivers? For instance when I run this locally, the SimpleRunner is on `debug` level, then the EventDetector is also on `debug` model but the ObjectTracker is on `info`.